akka-docs/rst/java/stream/stream-customize.rst

```diff
@@ -50,7 +50,7 @@ override ``onPull()`` which indicates that we are free to emit a single element.
 ``onDownstreamFinish()`` which is called if the downstream cancelled. Since the default behavior of that callback is
 to stop the stage, we don't need to override it. In the ``onPull`` callback we simply emit the next number.
 
-Instances of the above :class:`GraphStage` are subclasses of ``Graph<SourceShape<Int>,Unit>`` which means
+Instances of the above :class:`GraphStage` are subclasses of ``Graph<SourceShape<Integer>,NotUsed>`` which means
```

Comment:
[This](http://doc.akka.io/docs/akka/2.4/scala/stream/stream-customize.html#Custom_processing_with_GraphStage) refers to [`GraphStageDocTest.java`](https://github.com/akka/akka/blob/4a7cd147313404eb8137e56163e54f06c23640d0/akka-docs/rst/java/code/docs/stream/GraphStageDocTest.java#L60)

```
public class NumbersSource extends GraphStage<SourceShape<Integer>>
```

where [`GraphStage`](https://github.com/akka/akka/blob/4a7cd147313404eb8137e56163e54f06c23640d0/akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala#L42) is defined as follows:

```
abstract class GraphStage[S <: Shape] extends GraphStageWithMaterializedValue[S, NotUsed] {
```